src/chains/mainnet/vm/opcodes/index.ts

```diff
@@ -0,0 +1,6 @@
+import { Opcode } from '@/chains/types';
+import { opcodes as arithmeticOpcodes } from './arithmetic';
+import { opcodes as blockOpcodes } from './block';
+import { opcodes as memoryOpcodes } from './memory';
+
+export const opcodes: Opcode[] = [...arithmeticOpcodes, ...blockOpcodes, ...memoryOpcodes];
```

Comment:
Instead of exporting an array, it might be easier to export an object where the key is the opcode number. This will make it easier to look up opcodes by number, and will also make it easier to replace opcodes, e.g. `optimismOpcodes = { ...mainnetOpcodes, 0x41: { ... } }` to just replace opcode number `0x41`

Reply:
I agree and it's also easier to read the code now :)
Fixed by [db0ecf1](https://github.com/mds1/evm-diff/pull/12/commits/db0ecf1ba4eb4d8f7313406832ad14199922e625)